src/web3-utils.js

```diff
@@ -145,13 +146,18 @@ const gasPriceApi = 'https://ethgasstation.info/json/ethgasAPI.json'
 export async function getGasPrice({
   mainnet: { safeMinimum = '3', disableEstimate } = {},
 } = {}) {
-  if (network.type !== 'main') {
-    // Hardcode 10 for non-mainnet networks
+  if (network.chainId === CHAIN_IDS.XDAI) {
```

Comment:
There does not appear to be a public API providing gas price estimates on xDai yet... which is probably fine given there are almost no transactions.